hypertools/tools/cluster.py

```diff
@@ -1,13 +1,117 @@
 #!/usr/bin/env python
 import warnings
 from sklearn.cluster import KMeans, MiniBatchKMeans, AgglomerativeClustering, Birch, FeatureAgglomeration, SpectralClustering
+from sklearn.mixture import GaussianMixture, BayesianGaussianMixture
+
 import numpy as np
 from .._shared.helpers import *
 
+
+class Cluster(object):
+    """ 
+        Cluster Class holds generic attributes
```

Comment:
this docstring is not in line with the quotes

Reply:
Has been aligned now.